CppCoreGuidelines.md

```diff
@@ -6191,7 +6189,6 @@ After `y = std::move(x)` the value of `y` should be the value `x` had and `x` sh
         X x{};
         // ...
         X y = std::move(x);
-        x = X{};   // OK
```

Comment:
I think this was there to demonstrate what can be done to a moved-from object
(presumably op= is in the `// ...` part of X)